pkg/controllers/user/logging/projectlogging.go

```diff
@@ -21,6 +22,9 @@ const (
 	ProjectIDAnnotation = "field.cattle.io/projectId"
 )
 
+// ProjectLoggingSyncer listens for projectLogging CRD in management API
+// and update the changes to configmap, deploy fluentd
+
 type ProjectLoggingSyncer struct {
 	management           config.ManagementContext
```

Comment:
We can remove this field since there's no reference to it.

Reply:
ok